.github/workflows/dependabot_auto_merge.yaml

```diff
@@ -0,0 +1,22 @@
+name: Dependabot auto-merge
+
+on:
+  workflow_run:
+    types:
+      - completed
+    workflows:
+      - Continous Integration and Demo Deployment
+
+jobs:
+  auto-merge:
+    name: Auto-merge
+    runs-on: ubuntu-latest
+    if: ${{ github.actor == 'dependabot[bot]' }}
+    steps:
+      - name: Auto-merge
+        if: ${{ github.event.workflow_run.conclusion == 'success' }}
+        uses: ridedott/merge-me-action@v2
```

Comment:
@nelsonkopliku this ensures that we run this workflow only on successful conclusion of the CI workflow

Reply:
ah, thanks :heart: